src/app/Core/ExportServices/objectsToZIP.service.ts

```diff
@@ -6,7 +6,7 @@ import { generateParameterHeader } from "./generateParameterHeader.service";
 import { statDataStringGenerator } from "./generateStatString.service";
 import { RAWDataStringGenerator } from "./generateRAWString.service";
 
-interface ObjectToCsvBothProps {
+interface ObjectToZIPProps {

```

Comment:
same here: Zip instead of ZIP

Reply:
Done. Commit "Change ZIP/CSV to Zip/CSV "